meta/peg.py

```diff
@@ -68,15 +68,15 @@ def at(self, offset):
         return self.tokens[self.position + offset]
 
     def read_token(self):
-        #print self.at(0)
+        #print(self.at(0))
         self.position += 1
         return self.at(-1)
 
     def read_toplevel(self):
         name = self.read_token()
         equals = self.read_token()
 
-        assert all(ident_char(c) for c in name)
+        assert all([ident_char(c) for c in name])
```

Comment:
Adding the square brackets here shouldn't be necessary - Python 2 and 3 both support generator expressions.

Same thing applies in other places where you've added them.

Reply:
Thanks, I added them due to running into errors but I think that was just a function of a parser fallback which is no longer happening due to other changes. Fixed